Chapters/1-1-Chapter1.md

```diff
@@ -0,0 +1,167 @@
+## Chapter 1
+
+_It's quiet. Something is wrong._
+
+_No. It's fine. Just a little late._
+
+_It's never this late._
+
+The wind picked up and Ara was warming the land, but still the ziyu tree remained silent. I took a deep breath and examined it, but nothing looked different: it was still the same tree I remembered my mother taking me to.
+
+_It must be the wind. The wind is too dry right now._
+
+Patiently, I waited for an answer that would settle my mind, but none came. There would be no song today.
+
+I stood up and dusted off my trousers, catching a last glimpse of the twilight-pink sky through the quiet leaves. I felt the presence of some malicious spirit, brooding within the very breeze I drew my breaths from.
+
+"The wind speaks," my mother would say, "The ziyu tells us what it's saying."
+
+"What is it saying?" I must have asked hae a thousand times. My mother would tell me to listen a little closer, that the answer was not something I could hear. I knew that the silence was an answer, but the trouble was in the question.
+
+"Lovely morning, Liffa."
+
+Startled, I sat up to see a silhouette of Melna standing in the light, her hair dancing in time with the leaves.
+
+"How long have you been here?" I asked.
+
+"Just a moment. I figured you would be here today."
+
+I stayed silent, still listening for the leaves and my mother's voice.
+
+"You've been gone a while. Breakfast is ready soon."
+
+"I'll be there in a bit."
+
+She stared at me for a moment, I assume to size up my strange behavior.
+
+"There's a chill in the air," she remarked, "Don't stay out too long."
+
+---
+
+Kipnu was at the stove reheating a sweet mash of pohua with his morning face on. His hair resembled the furrowed nest of a bird that took a liking to the most crooked of branches. He looked up, locking his tired eyes to mine, and put on his laziest morning smile which only complimented his hair. I walked up to hug and nuzzle him good morning. Melna was squatting at a table in the great room, reading her daily missive of provincial news.
```

Comment:
> Kipnu was at the stove reheating a sweet mash of pohua with his morning face on.

"Subject A - Doing B - Modifier for subject A" reads awkwardly because you are going A B A. Especially since the sentence has two new words in it that the reader has to learn.

What about? "With his morning face on, Kipnu reheated a sweet mash of pohua over the stove."

If you introduce Kipnu by description first the reader can have a familiar idea to latch on to. Whether it's grumpy or energetic "a morning face" is something everyone knows, it leads them to learn about this new character. And the rest of the sentence is fine. No idea what a pohua is, but that doesn't matter right now. It gives me an accurate visual of their lifestyle.

Reply:
I think you're right. I've removed pohua, and just said he's standing over the stove. Also made him kind of a standard hunk/expected hero.